manim/utils/tex.py

```diff
@@ -1,298 +1,179 @@
-            return self.generate_tex_command(
-                "usepackage", required_params=[pkg[0]], optional_params=pkg[1]
-            )
-        else:
-            return self.generate_tex_command("usepackage", required_params=[pkg])
-
-    def get_text_for_text_mode(self, expression):
-        """Inserts an expression verbatim into the TeX template.
+    def get_texcode_for_expression(self, expression):
+        """Inserts expression verbatim into TeX template.
 
         Parameters
         ----------
         expression : :class:`str`
-            The expression to be typeset, e.g. ``"foo"``.
+            The string containing the expression to be typeset, e.g. ``$\\sqrt{2}$``
 
         Returns
         -------
         :class:`str`
-            LaTeX code based on the template, containing the given expression and ready for typesetting
+            LaTeX code based on template, containing the given expression and ready for typesetting
         """
-        return self.body.replace(self.text_to_replace, expression)
+        return self.body.replace(self.placeholder_text, expression)
 
-    def get_text_for_env(self, environment, expression):
-        """Inserts an expression wrapped in a given environment into the TeX template.
+    def get_texcode_for_expression_in_env(self, expression, environment):
+        """Inserts expression into TeX template wrapped in \begin{environemnt} and \end{environment}
 
         Parameters
         ----------
-        environment : :class:`str`
-            The environment in which we should wrap the expression.
         expression : :class:`str`
-            The string containing the expression to be typeset, e.g. ``"$\\sqrt{2}$"``
+            The string containing the expression to be typeset, e.g. ``$\\sqrt{2}$``
+        environment : :class:`str`
+            The string containing the environment in which the expression should be typeset, e.g. ``align*``
 
         Returns
         -------
         :class:`str`
-            LaTeX code based on template, containing the given expression and ready for typesetting
+            LaTeX code based on template, containing the given expression inside its environment, ready for typesetting
         """
         begin = r"\begin{" + environment + "}"
         end = r"\end{" + environment + "}"
         return self.body.replace(
-            self.text_to_replace, "{0}\n{1}\n{2}".format(begin, expression, end)
+            self.placeholder_text, "{0}\n{1}\n{2}".format(begin, expression, end)
         )
 
-    def get_text_for_tex_mode(self, expression):
-        """Inserts an expression within an ``align*`` environment into
-        the TeX template.
 
-        Parameters
-        ----------
-        expression : :class:`str`
-            The string containing the (math) expression to be typeset,
-            e.g. ``"$\\sqrt{2}$"``
+class TexTemplateFromFile(TexTemplate):
+    """A TexTemplate object created from a template file (default: tex_template.tex)"""
```

Comment:
attributes and init params

Reply:
These are all resolved, again by a new commit (https://github.com/ManimCommunity/manim/pull/483/commits/79d0d7c9da88b831920ed42a5e97962ab5615796)